src/ORM/DataList.php

```diff
@@ -1367,6 +1421,46 @@ private function addEagerLoadedDataToParent(
         }
     }
 
+    private function fillEmptyEagerLoadedRelations(
+        Query|array $parents,
+        array $parentIDs,
```

Comment:
```suggestion
        array $missingParentIDs,
```

Reply:
These aren't missing.... they're known IDs that don't have anything in their relation. But not worth discussing IMO so I've made the change.